src/decorators/bind.ts

```diff
@@ -170,7 +170,7 @@ function scheduleIsComponentCheck(binding: BindSuperConfig<unknown>) {
     try {
       checkIsComponent(binding.componentProto);
     } catch (ex) {
-      const target = binding.all ? JSON.stringify(binding.all) : binding.targetPath.join('.');
+      const target = binding.all ? JSON.stringify(binding.all) : binding.targetPath.slice(1).join('.');
```

Comment:
My initial expectation would be that the binding path would not include the direction of the binding? Would this be easier to understand if this property had another name, e.g. `binding.spec` or something like that?

Reply:
can't really change that without completely re-doing the API